template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Views/Shared/Error.cshtml

```diff
@@ -12,3 +13,9 @@
 <p>
     <strong>Development environment should not be enabled in deployed applications</strong>, as it can result in sensitive information from exceptions being displayed to end users. For local debugging, development environment can be enabled by setting the <strong>ASPNETCORE_ENVIRONMENT</strong> environment variable to <strong>Development</strong>, and restarting the application.
 </p>
+@if (Activity.Current != null)
+{
+<p>
+    <strong>Correlation ID:</strong> @Activity.Current.Id
```

Comment:
Since there are word and non-word characters, selecting it with double-click is not possible (at least in chrome, may be there is a difference with other browsers) 
E.g. try to select |abc1def2-12345.f123_1.

Quotes just emphasis that all the text including the start and end characters are part of the Id.